sdk/schemaregistry/azure-data-schemaregistry-apacheavro/src/test/java/com/azure/data/schemaregistry/apacheavro/SchemaRegistryApacheAvroSerializerIntegrationTest.java

```diff
@@ -62,29 +53,19 @@ public class SchemaRegistryApacheAvroSerializerIntegrationTest extends TestProxy
     static final String PLAYBACK_TEST_GROUP = "azsdk_java_group";
     static final String PLAYBACK_ENDPOINT = "https://foo.servicebus.windows.net";
 
-    private TokenCredential tokenCredential;
     private String schemaGroup;
     private SchemaRegistryClientBuilder builder;
-    private String endpoint;
     private String eventHubName;
     private String connectionString;
 
     @Override
     protected void beforeTest() {
+        String endpoint;
+        TokenCredential tokenCredential;
         if (interceptorManager.isPlaybackMode()) {
-            tokenCredential = mock(TokenCredential.class);
+            tokenCredential = new MockTokenCredential();
```

Comment:
What was the cause of the test failures? Was mockito incorrectly mocking this?

Reply:
The mock was fine, this is just a better version than the mock as we don't care about the credential response during playback other than it didn't throw an error. This change was just a piece of reducing places where we use Mockito as there is a lot of baggage with it, especially given Mockito is no longer actively developing with a Java 8 baseline.